mmv1/third_party/terraform/website/docs/r/google_project_iam.html.markdown

```diff
@@ -153,6 +155,8 @@ resource "google_project_iam_audit_config" "project" {
 
 The following arguments are supported:
 
+~> **Note:** For `member/members` argument, `google_project_iam_binding` expects `members` field while `google_project_iam_member` expects `member` field.
```

Comment:
Can we work this into the field description instead?

Reply:
I've added this in the `member/members` field description